app/views/layouts/application.html.haml

```diff
@@ -27,12 +26,12 @@
               = link_to "Create a publication", new_publication_path, class: 'nav-link'
             %li.nav-item.active
               = link_to "Publications", publications_path, class: 'nav-link'
-            %li.nav-item
+            %li.nav-item.active
```

Comment:
why are all active? make all as not active. active is when they are currently on it.